moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_world_distance_field.h

```diff
@@ -143,12 +145,12 @@ class CollisionWorldDistanceField : public CollisionWorld
   void distanceRobot(const DistanceRequest& req, DistanceResult& res, const CollisionRobot& robot,
                      const robot_state::RobotState& state) const override
   {
-    ROS_ERROR_NAMED("collision_distance_field", "Not implemented");
+    RCLCPP_ERROR(LOGGER_COLLISION_DISTANCE_FIELD, "Not implemented");
```

Comment:
This class should have it's own logger. Perhaps 
`static rclcpp::Logger LOGGER_COLLISION_WORLD_DISTANCE_FIELD = LOGGER_COLLISION_DISTANCE_FIELD.get_child("collision_world_distance_field")`

Reply:
https://github.com/ros-planning/moveit2/pull/65#discussion_r293258198